tests/test_identity_linking.py

```
@@ -0,0 +1,191 @@
+
+    row = storage.get_identity_link("d1", "octocat")
+    assert row is not None
+    assert row["verified"] == 1
+
+
+def test_verify_marks_mapping_verified_and_clears_code(tmp_path: Path) -> None:
+    storage = SqliteStorage(data_dir=str(tmp_path))
+    storage.init_schema()
+
+    svc = IdentityLinkService(storage=storage, github_identity=_GitHubIdentityAlways(True, "bio"))
+    claim = svc.create_claim("d1", "octocat")
```

Ruff (0.14.14)

[error] Local variable `claim` is assigned to but never used

Remove assignment to unused variable `claim`

(F841)